static/app/stories/storyBook.tsx

```diff
@@ -30,9 +30,7 @@ export default function storyBook(
   const apiReferenceFn: (
     documentation: TypeLoader.ComponentDocWithFilename | undefined
   ) => void = (documentation: TypeLoader.ComponentDocWithFilename | undefined) => {
-    if (documentation) {
```

Comment:
This change will also ensure that components that have API Reference sections but are not run in correct storybook mode display this fallback message instead of not showing anything

![CleanShot 2025-02-20 at 17 55 15@2x](https://github.com/user-attachments/assets/aa6a358a-62a1-4fe5-8465-583c4682cf0d)